doc/ssl/SSL_CIPHER_get_name.pod

```diff
@@ -14,6 +14,10 @@ SSL_CIPHER_get_name, SSL_CIPHER_get_bits, SSL_CIPHER_get_version, SSL_CIPHER_des
  char *SSL_CIPHER_description(const SSL_CIPHER *cipher, char *buf, int size);
  int SSL_CIPHER_get_cipher_nid(const SSL_CIPHER *c);
  int SSL_CIPHER_get_digest_nid(const SSL_CIPHER *c);
+ int SSL_CIPHER_get_kx_nid(const SSL_CIPHER *c);
+ int SSL_CIPHER_get_auth_nid(const SSL_CIPHER *c);
+ int SSL_CIPHER_get_fips_nid(const SSL_CIPHER *c);
```

Comment:
We (Akamai) don’t need is_FIPS.  And since we don't do FIPS in 1.1, it could just be ```#define is_FIPS() 0```